src/queries/proposals.ts

```diff
@@ -32,6 +32,13 @@ export const getAllProposalsbyServiceId = (id: string): Promise<any> => {
           rating
           numReviews
         }
+        description {
+          title
```

Comment:
I think we need some king of tag extension to use fragment. 
Here is one example :  https://www.npmjs.com/package/graphql-tag
I'm not sure if it's mandatory, I need more research. 
It could be interesting to research also on -> https://github.com/dotansimha/graphql-code-generator 

For the fragment I'll just use some variable for now.